schemas/JoinedRoom.json

```diff
@@ -15,7 +15,9 @@
        {"const": "RoomDirectory", "description": "Room joined via the public rooms directory."},
        {"const": "SpaceHierarchy", "description": "Room joined via the space hierarchy view."},
        {"const": "Timeline", "description": "Room joined via a timeline pill or link in another room."},
-       {"const": "Notification", "description": "Room joined via a push/desktop notification."}
+       {"const": "Notification", "description": "Room joined via a push/desktop notification."},
+       {"const": "MobilePermalink", "description": "Room joined via link"},
```

Comment:
This one could be `Timeline` to match ViewRoom

Reply:
Link is not always opened from timeline, It can also be triggered by clicking on link in another application for example